db/migrate/20190201211125_backport_default_vettec_to_education_benefits_submissions.rb

```diff
@@ -0,0 +1,8 @@
+class BackportDefaultVettecToEducationBenefitsSubmissions < ActiveRecord::Migration
+  def change
+    EducationBenefitsSubmission.select(:id).find_in_batches.with_index do |records, index|
```

Comment:
This should be a temporary rake task: https://thoughtbot.com/blog/data-migrations-in-rails

Reply:
oh I missed that zero down time said do it in a migration 🤔 I still think it should be a temp rake task because it doesn't need to live forever in migrations.

Reply:
Well perhaps if a rollback were necessary, then rails migration magic would know how to undo this operation 🤷‍♂️

Reply:
In short, I agree that a temporary rake task is best.

Reply:
Yep - good catch. Should PR that gem! 😄

Reply:
file has been deleted